src/pages/settings.tsx

```diff
@@ -1,17 +1,46 @@
 import React from "react";
-import ProgressBar from "@/components/design_system/progressBar/ProgressBar";
+import StepperCustom from "@/components/design_system/stepper/StepperCustom";
+import Box from "@mui/material/Box";
+import Stepper from "@mui/material/Stepper";
+import Step from "@mui/material/Step";
+import StepLabel from "@mui/material/StepLabel";
+import CheckCircleIcon from "@mui/icons-material/CheckCircle";
+import TripOriginRoundedIcon from "@mui/icons-material/TripOriginRounded";
 
 const Settings = () => {
   // TODO? make sure settings displayed reflect the specific logged-in user
 
+  const currentStep = 1;
+  const steps = ["Completed", "Active", "Disabled"];
+
   return (
     <div>
       <p>🚧 Under Construction! 🚧</p>
       <br />
-      <p>Demonstration of Progress Bar at 50%</p>
-      <p>{`(I will delete this from settings page when PR approved)`}</p>
+      <p>Demonstration of Stepper</p>
+      <p>{`(to be deleted later)`}</p>
       <br />
-      <ProgressBar value={50} />
+      <StepperCustom />
```

Comment:
You have both the StepperCustom component and a written out version of what looks very similar to StepperCustom. I know you will probably remove these from the settings file, but double-check to make sure that the StepperCustom is the final version and not this written out version.